js/state/siteUtil.js

```diff
@@ -219,10 +219,10 @@ const mergeSiteDetails = (oldSiteDetail, newSiteDetail, tag, folderId, order) =>
  * @param tag The tag to add for this site
  *   See siteTags.js for supported types. No tag means just a history item
  * @param originalSiteDetail If specified, use when searching site list
- * @param {Function=} syncCallback specified if this change should be synced
+ * @param skipSync If specified, this item is not eligible for sync
```

Comment:
Yeah, I think actually we should rename it to something clearer like `addedBySync` since it is just keeping track of which records were downloaded by sync.